recipes/trimal/1.4/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: trimal
+  version: "1.4"
+
+source:
+  fn: trimAl.tar.gz
+  url: https://github.com/scapella/trimal/archive/trimAl.tar.gz
```

Comment:
A branch is also not a fixed version, the developer can check in some new stuff and you will get it with a rebuild.
Why not this one: https://github.com/scapella/trimal/archive/v1.4.1.tar.gz -- to old?